.ddev/config.yaml

```diff
@@ -26,13 +26,15 @@ web_environment:
   - ELASTICSEARCH_HOST=elasticsearch
   - VARNISH_ADMIN_HOST=varnish
   - VARNISH_ADMIN_PORT=80
+  - SMTP_ADDRESS=localhost:1025
 
 corepack_enable: false
 hooks:
   post-start:
     - exec: "composer install"
+    - exec: "npm ci"
     - exec: |
-        plugins=(analysis-icu analysis-ukrainian)
+        plugins=(analysis-icu)
```

Comment:
Related to this change, on a fresh install I get this message:
```
-> Installing analysis-icu
-> Downloading analysis-icu from elastic
[=================================================] 100%??
-> Installed analysis-icu
-> Please restart Elasticsearch to activate any plugins installed
```

Should we also restart elasticsearch after the installation of the plugin somehow?